common/src/main/java/org/apache/celeborn/common/client/MasterClientWithRetry.java

```diff
@@ -48,8 +48,8 @@
 import org.apache.celeborn.common.rpc.RpcTimeout;
 import org.apache.celeborn.common.util.ThreadUtils;
 
-public class RssHARetryClient {
-  private static final Logger LOG = LoggerFactory.getLogger(RssHARetryClient.class);
+public class MasterClientWithRetry {
```

Comment:
No strong option, but I would call it `MasterClient` directly. What do you think? @waitinfuture @FMX

Reply:
> No strong opinion, but I would call it `MasterClient` directly. What do you think? @waitinfuture @FMX

+1

Reply:
Done